lib/src/header_parser/type_extractor/cxtypekindmap.dart

```diff
@@ -27,6 +28,22 @@ var cxTypeKindToSupportedNativeTypes = <int, SupportedNativeType>{
   clang.CXTypeKind.CXType_Char_U: SupportedNativeType.Uint8,
 };
 
+var cxTypeKindToImportedTypes = <String, ImportedType>{
+  'void': voidType,
+  'unsigned char': unsignedCharType,
+  'char': charType,
```

Comment:
+= 'signed char' (I updated the package:ffi PR)

Reply:
Done